libs/ui/search/src/lib/results-table/results-table.component.ts

```diff
@@ -103,6 +99,10 @@ export class ResultsTableComponent {
     this.duplicateRecord.emit(item as CatalogRecord)
   }
 
+  handleDelete(item: unknown) {
```

Comment:
Can the type be set here already as `CatalogRecord`? I see it's also unknown in the other functions, but was wondering if that could be changed.

Reply:
I just tried again, but the table component is generic (it can work with items of any types as long as the data matches the template), so the row is of unknown type in the event.